### DISCUSSION TITLE
From icelake to Iceberg Rust

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#426] [Announcements]
# From icelake to Iceberg Rust

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2024/05-from-icelake-to-iceberg-rust/

**Comment by trungda** [comment]
I would love to contribute. How do I start?

**Reply by Xuanwo** [reply]
Welcome to checkout https://github.com/apache/iceberg-rust/contribute
